### ISSUE TITLE
can't access property "increment", firestore.FieldValue is undefined

### ISSUE THREAD

**Issue by SwiftWinds** [FirebaseExtended/reactfire#268]
### Version info

**React:** 16.13.1

**Firebase:** 7.15.5

**ReactFire:** 2.0.3

**Node:** v12.18.1

### Test case

https://stackblitz.com/edit/react-nctwiy

### Steps to reproduce

Wrap your `<App />` in `FirebaseAppProvider`, set `const firestore = useFirestore()`, set `const increment = firestore.FieldValue.increment(1)`, and watch everything implode ;P 

### Expected behavior

There not to be an error

### Actual behavior

`can't access property "increment", firestore.FieldValue is undefined`

### Additional details

Thanks for the awesome library, guys! :) It helps accelerate Firebase in React by a lot and makes development feel like a breeze.

**Comment by prescottprue** [comment]
You are attempting to use the statics (i.e. `firebase.firestore` instead of `firebase.firestore()`). Try doing the following instead:

```js
const firestoreStatics = useFirestore
firestoreStatics.FieldValue.increment(1)
```

or you can destructure:

```js
const { FieldValue } = useFirestore
FieldValue.increment(1)
```

**Comment by SwiftWinds** [comment]
Yup! That seems to have done the trick; thanks! :)

**Comment by SwiftWinds** [comment]
> This should probably be more clearly covered in the docs though

Yeah, I would tend to agree. It's just that one of the resources that I found on ReactFire (and how I found out about it in the first place) was from [this talk](https://youtu.be/Mi9aKDcpRYA?t=1103), where he does:
```js
const firestore = useFirestore();
...
const increment = firestore.FieldValue.increment(1);
```
and it somehow works! 🤨 I guess this talk was some time ago, though, so perhaps the API changed.

Anyways, although the [Firebase API Reference](https://firebase.google.com/docs/reference/js/) went a long way to help understand Firebase, more specifics on the difference between a firebase app object, a static `firebase.firestore`, and a non-static `firebase.firestore()` would help greatly.
